charts/bluehub/v0.1.0/questions.yaml

```diff
@@ -0,0 +1,261 @@
+labels:
+ io.rancher.certified: partner
+questions:
+- variable: zookeeper.enabled
+  default: "false"
+  type: boolean
+  label: "If true, installs the Zookeeper"
```

Comment:
Can you update all of these questions to have a short brief `label` like "Install Zookeeper" and use the `description` key for more detailed descriptions of what the value does?

Reply:
changes made as suggested